django/cantusdb_project/main_app/templates/chant_create.html

```diff
@@ -255,11 +255,59 @@ <h3>Create Chant</h3>
             <div class="search-bar mb-3">
                 {% include "global_search_bar.html" %}
             </div>
-            <div class="card w-100">
+
+            <div class="card w-100 mb-3">
                 <div class="card-header">
-                    <h5><a id="source" href="{% url 'source-detail' source.id %}">{{ source.title }}</a></h5>
+                    <h5><a id="source" href="{% url 'source-detail' source.id %}">{{ source.siglum }}</a></h5>
```

Comment:
Is this intentional?

Reply:
@annamorphism was looking over my shoulder as I did a bit of wireframing. This, iirc, is to make it match the sidebar cards on other pages.

But it should have been in its own commit. Thanks for checking on this.

@annamorphism, all this sounds right, yes?

Reply:
Ok! That makes sense... I really was just making sure it didn't slip in unintentionally.

Reply:
yes--OldCantus had the full title here (for some reason), which is probably why this got a title as well, but all the other similar sidebar cards use the siglum and we decided to make it match (better use of space, too.)

Reply:
Perfect! Going to "resolve"...